### ISSUE TITLE
=Bad file= IMC from other mod

### ISSUE THREAD

**Issue by jacobmix** [SleepyTrousers/EnderIO#4582]
#### Issue Description:
Me, and my friends have a "modpack" with around 300 mods (ran fine for a while)
After updating some mods, and added a few i seem to get an error now that i can't pin point how to fix.
I wanna know if there's a way to keep all the mods, but just change a config to make it not crash.

#### What happens:
Game crashes when it loads the Minecraft main menu after all the mods have loaded.
This is the error i get: http://prntscr.com/j8kujg

#### What you expected to happen:
For it to not crash, and load the menu with no error.

#### Steps to reproduce:

1. Install custom mod pack: https://mega.nz/#!CIFChZyC!dtBQHBYV2xL8BH2OFuOd30Ld3lDeRivB5TBCg8IECyM
2. When it's done installing launch it, and wait for it to finish.
3. After 10 years pass you're greeted with an error message. (prinscr.com's screenshot above)

____
#### Affected Versions:
- EnderIO: 1.12.2-5.0.22
- EnderCore: 1.12.2-0.5.20
- Minecraft: 1.12.2
- Forge: 14.23.3.2670

#### Your most recent log file where the issue was present: 
Log: https://pastebin.com/RYuM75iA
Debug log: https://pastebin.com/aWcWtB6H

**Comment by tyler489** [comment]
A modslist would be helpful. But do you have the charcoal pit mod Overview - Charcoal Pit - Mods - Projects - Minecraft CurseForge
https://minecraft.curseforge.com › projects

**Comment by tyler489** [comment]
Also did you add any custom recipes?

**Comment by HenryLoenwind** [comment]
The next build of Ender IO will no longer suppress the log message about receiving IMC messages.

**Comment by MokahTGS** [comment]
Rolling back  JOAPCA to 2.2.6.78 fixes the issue.

**Comment by tyler489** [comment]
Roll back  to jaopcs  v.78 for now

**Comment by jacobmix** [comment]
@tyler489 Yeah. I've just updated all the mods in the pack, and added even more :p 
Launches fine, and plays great... But I seem to crash randomly tho.
Just goes "not responding" and everyone on my "lan" hosted server get's disconnected.
It couldn't be because having 300+ mods is a bad idea. Surely not.
If you wanna test out the crazy mod pack you can get it here:
https://mega.nz/#!aJcAlTKI!7NSPyV4sEIxYJxa0pPDt_aLJD4n5PCXEWALuc4WjhtM
Here's my crash logs, ect:
Crash report: https://pastebin.com/Cp5MJaw4
Latest log: https://pastebin.com/x6LvSPrE
Debug log split into 3 parts.
1: https://pastebin.com/BGK3BjBy 
2: https://pastebin.com/24RBX0R7
3: https://pastebin.com/invC9t8m
My specs in form of a DxDiag.txt file: https://pastebin.com/6eJLjA2r

**Comment by tyler489** [comment]
Something is messing with world entity spawner

**Comment by EpicSquid** [comment]
This issue is closed and has nothing to do with enderio

**Comment by jacobmix** [comment]
@EpicSquid Yeah it's a fixed, and closed issue. Just talking about my modpack, and how it is after updates.
What's the issue here? You shouldn't see this if you're searching for current issues.
I'm just posting things related to the issue i had (the modpack as a whole) there shouldn't be a problem.
If this actually causes any problem just say. For now if you want an actual issue go to the opened ones

@tyler489 I'll see if i can figure it out. Using 300+ mods isn't as easy as just adding them all sadly.
Just a note. The script, and config files are separate from the mod pack download.
Me, and my friends are still changing them, and upload changes regularly. Thanks for the help tho.

**Comment by EpicSquid** [comment]
Something is wrong with vanilla spawners trying to find a chunk so spawn in

**Comment by jacobmix** [comment]
Tried using some of the scripts from DireWolf20's mod pack, and i haven't crashed for a while.
Gonna have to look into what fixes it. Don't wanna change recipes or how some mods work too much.
